tools/ui-components/tsconfig.json

```
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "module": "ES6",
+    "target": "ES6",
+    "sourceMap": true,
+    "jsx": "react",
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "moduleResolution": "node"
```

```suggestion
    "moduleResolution": "node",
    "strict": "true",
    "noEmit": "true"
```
`noEmit` is necessary if we want to avoid compiling when we're just type-checking.